CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -273,25 +288,76 @@ private static void OnApplicationExit(object sender, EventArgs e)
         /// <summary>
         /// Initializes a new instance of the <see cref="ChromiumWebBrowser"/> class.
         /// </summary>
+        public ChromiumWebBrowser()
+        {
+
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChromiumWebBrowser"/> class.
+        /// </summary>
         /// <param name="address">The address.</param>
-        /// <exception cref="System.InvalidOperationException">Cef::Initialize() failed</exception>
         public ChromiumWebBrowser(string address)
         {
-            if (!Cef.IsInitialized && !Cef.Initialize())
+            Address = address;
+        }
+
+        private void Initialize()
+        {
+            if (!initialized)
             {
-                throw new InvalidOperationException("Cef::Initialize() failed");
-            }
+                if (!Cef.IsInitialized && !Cef.Initialize())
+                {
+                    throw new InvalidOperationException("Cef::Initialize() failed");
+                }
 
-            Cef.AddDisposable(this);
-            Address = address;
+                Cef.AddDisposable(this);
+
+                if (FocusHandler == null)
+                {
+                    FocusHandler = new DefaultFocusHandler(this);
+                }
+
+                if (ResourceHandlerFactory == null)
+                {
+                    ResourceHandlerFactory = new DefaultResourceHandlerFactory();
+                }
 
-            Dock = DockStyle.Fill;
+                if (BrowserSettings == null)
```

Comment:
Moving this opens up the potential for null references exceptions.

Previously you could write `browser.BrowserSettings.WhateverProperty = value`

I understand the reason for moving this, just needs to be documented. Probably no point setting it in a delayed fashion.